br/pkg/streamhelper/daemon/interface.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 PingCAP, Inc. Licensed under Apache-2.0.
+
+package daemon
+
+import "context"
+
+// Interface describes the lifetime hook of a daemon application.
+type Interface interface {
```

Comment:
can we have a better name?

Reply:
This name style is ported from [`sort.Interface`](https://pkg.go.dev/sort@go1.19.1#Interface), in some way, I guess using the name `Interface` to name the main interface of the package is pretty go-style. 🤔

Reply:
OK